federation/pkg/kubefed/unjoin_test.go

```diff
@@ -184,9 +184,9 @@ func testUnjoinFederationFactory(name, server, secret string) cmdutil.Factory {
 	tf.ClientConfig = kubefedtesting.DefaultClientConfig()
 	ns := serializer.NegotiatedSerializerWrapper(runtime.SerializerInfo{Serializer: runtime.NewCodec(f.JSONEncoder(), api.Codecs.UniversalDecoder(fedv1beta1.SchemeGroupVersion))})
 	tf.Client = &fake.RESTClient{
-		APIRegistry:          api.Registry,
+		GroupVersion:         api.Registry.GroupOrDie("federation").GroupVersion,
+		InternalGroupName:    api.Registry.GroupOrDie("federation").GroupVersion.Group,
```

Comment:
do we have cases with distinct internal and external group names? I think we always register the internal types into all relevant internal group versions.

Reply:
I don't think so, but figured I'd let them be explicitly separate here since they were previously controlled separately.  I can change that if you want.

Reply:
It simplifies the interface.  So why not.